app/src/main/java/com/hedvig/app/feature/insurance/ui/Subheading.kt

```diff
@@ -1,20 +1,23 @@
 package com.hedvig.app.feature.insurance.ui
 
+import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
 import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import com.hedvig.app.ui.compose.theme.HedvigTheme
 
 @Composable
-fun Subheading(text: String) {
-    Text(
-        text = text,
-        style = MaterialTheme.typography.h6,
+fun Subheading(
+    text: String,
+    contentBeforeText: (@Composable () -> Unit)? = null,
```

Comment:
```suggestion
    contentBeforeText: (@Composable () -> Unit)? = null,
```

I think we're probably better off just separating these two, the API for `Subheading` became a bit needlessly complex 😬